<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="1" name="Failed suite [Surefire suite]" verbose="0">
  <test thread-count="1" name="Surefire test(failed)" verbose="0">
    <classes>
      <class name="com.fortune.testcases.LoginTest">
        <methods>
          <include name="loginByValidCredentials"/>
          <include name="loginByBlankCredentials"/>
          <include name="launch"/>
          <include name="generateReport"/>
          <include name="closed"/>
          <include name="loginByInvalidCredentials"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.fortune.testcases.LoginTest -->
      <class name="com.fortune.OutboundTask.OutBoundTaskTest">
        <methods>
          <include name="endTestMethod"/>
          <include name="startTestMethod"/>
          <include name="Login"/>
          <include name="openOutBoundTaskMenu"/>
          <include name="generateReport"/>
          <include name="executeTelecallingTestCases"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- com.fortune.OutboundTask.OutBoundTaskTest -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
